flutter_local_notifications/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -39,5 +39,12 @@
         <meta-data
             android:name="flutterEmbedding"
             android:value="2" />
+        <service
+            android:name="com.dexterous.flutterlocalnotifications.ForegroundService"
+            android:enabled="true"            
+            android:exported="false"
+            android:stopWithTask="true"
```

Comment:
Are you talking about line 46 (github seems to not show me the exact line here...)? 

The reason I added the `android:stopWithTask` attribute here is that in my opinion, it is very hard to find documentation about it, since the official [service tag documentation](https://developer.android.com/guide/topics/manifest/service-element) doesnt mention it and I wanted to give developers a hint what they need to search for to eventually find it documented in [R.attr](https://developer.android.com/reference/android/R.attr#stopWithTask).

>  The docs right now currently say this should left to `true`

Do you have a link for that? The documentation I linked above says that the default value is `false`.

My suggestion would be to explicitly mention it but set it to it's default value `false`. Then, in an xml comment we could point towards the documentation:
```xml
            android:name="com.dexterous.flutterlocalnotifications.ForegroundService"
            android:enabled="true"            
            android:exported="false"
            <!-- If you want your foreground service stopped if your app is stopped, set this to true.
            See https://developer.android.com/reference/android/R.attr#stopWithTask -->
            android:stopWithTask="false"
```
What do you think?


Maybe a quick word to the other two attributes:
* If I think about it again, we should omit ` android:enabled="true" ` since this is the default value.
* In my opinion, only `android:exported="false"` makes sense: This service does not do any work and is just used by the app to gain foreground privileges. Why should it be started by an other app?

Reply:
Yes, I'm referring to line 46. Regarding the point that it was hard to fine, this is why I mentioned leaving it to documentation..

> Do you have a link for that? The documentation I linked above says that the default value is false.

Sorry I probably wasn't clear here but I was referring to the API docs you had for the `startForegroundService()` method https://github.com/EPNW/flutter_local_notifications/blob/2b0092843377e767cd63648c2145c3f1de71a790/flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart#L226. Here it's specified as `true` as opposed to being "As you like" like you have in the next line

Reply:
Forgot to add, yes, leaving it explicitly as false with a comment is fine and agree that `android:exported="false"` makes sense and it is what's being done on the receivers in the PR for Android 12 support https://github.com/MaikuB/flutter_local_notifications/pull/1175